feat: add support for NO_PROXY (#359)
semantic-release/gitlab: README.md

| `GL_URL` or `GITLAB_URL`       | The GitLab endpoint.                                      |
| `GL_PREFIX` or `GITLAB_PREFIX` | The GitLab API prefix.                                    |
| `HTTP_PROXY` or `HTTPS_PROXY`  | HTTP or HTTPS proxy to use.                               |

> | `NO_PROXY`                     | NO_PROXY bypass proxy hosts                               |

Reviewer: ```suggestion
| `NO_PROXY`                     | Patterns for which the proxy should be ignored. See [details below](#proxy-configuration). |
```

Author: Added

If your GitLab instance is exposed via plain HTTP (not recommended!) use `HTTP_PROXY` instead.

> #### Proxy bypass configuration
> 
> You can configure a bypass option by setting `NO_PROXY` environment variable: `NO_PROXY=*.host.com, host.com`
> 
> Depending on your gitlabUrl and a matching `NO_PROXY`it is bypassing your proxy configuration of the `HTTP_PROXY` or `HTTPS_PROXY` environment variable.
> 

Reviewer: Less is more :)

```suggestion
If you need to bypass the proxy for some hosts, configure the `NO_PROXY` environment variable: NO_PROXY=*.host.com, host.com`

```

Author: Changed in last commit

If your GitLab instance is exposed via plain HTTP (not recommended!) use `HTTP_PROXY` instead.

> If you need to bypass the proxy for some hosts, configure the `NO_PROXY` environment variable: NO_PROXY=*.host.com, host.com`

Reviewer: ```suggestion
If you need to bypass the proxy for some hosts, configure the `NO_PROXY` environment variable: `NO_PROXY=*.host.com, host.com`
```
